pkg/urbit/vere/io/ames.c

```diff
@@ -30,21 +33,57 @@
 */
   typedef struct _u3_ames {             //  packet network state
     u3_auto       car_u;                //  driver
+    u3_pier*      pir_u;                //  pier
     union {                             //
       uv_udp_t    wax_u;                //
       uv_handle_t had_u;                //
     };                                  //
-    c3_d          who_d[2];             //  identity
-    c3_o          fak_o;                //  fake keys
-    c3_s          por_s;                //  public IPv4 port
     c3_c*         dns_c;                //  domain XX multiple/fallback
     c3_d          dop_d;                //  drop count
     c3_d          fal_d;                //  crash count
     c3_w          imp_w[256];           //  imperial IPs
     time_t        imp_t[256];           //  imperial IP timestamps
     c3_o          imp_o[256];           //  imperial print status
+    c3_o          see_o;                //  can scry
+    c3_o          fit_o;                //  filtering active
+    c3_y          ver_y;                //  protocol version
+    c3_d          vet_d;                //  version mismatches filtered
+    c3_d          mut_d;                //  invalid mugs filtered
+    u3_panc*      pac_u;                //  packets pending forwards
+    c3_d          foq_d;                //  forward queue size
+    c3_d          fow_d;                //  forwarded count
+    c3_d          fod_d;                //  forwards dropped count
```

Comment:
These new counters should all be printed in `_ames_io_info()` (which is for SIGINFO/SIGUSR1).